specification/keyvault/data-plane/TypeSpec/routes.tsp

```diff
@@ -0,0 +1,520 @@
+*/
+@route("/certificates/{certificate-name}")
+@delete op deleteCertificate(
+        ...ApiVersionParameter,
+        /**
+        * The name of the certificate.
+        */
+        @path
+        "certificate-name": string
+): ArmResponse<DeletedCertificateBundle> | ErrorResponse;
+
+
+
+@summary("Sets the certificate contacts for the specified key vault.")
+/**
+* Sets the certificate contacts for the specified key vault. This operation requires the certificates/managecontacts permission.
+*/
+@route("/certificates/contacts")
+@put op setCertificateContacts(
+        ...ApiVersionParameter,
+        /**
+        * The contacts for the key vault certificate.
+        */
+        @body
+        "contacts": Contacts
+): ArmResponse<Contacts> | ErrorResponse;
+
+
+
+@summary("Lists the certificate contacts for a specified key vault.")
+/**
+* The GetCertificateContacts operation returns the set of certificate contact resources in the specified key vault. This operation requires the certificates/managecontacts permission.
+*/
+@route("/certificates/contacts")
+@get op getCertificateContacts(
+        ...ApiVersionParameter
+): ArmResponse<Contacts> | ErrorResponse;
+
+
+
+@summary("Deletes the certificate contacts for a specified key vault.")
+/**
+* Deletes the certificate contacts for a specified key vault certificate. This operation requires the certificates/managecontacts permission.
+*/
+@route("/certificates/contacts")
+@delete op deleteCertificateContacts(
+        ...ApiVersionParameter
+): ArmResponse<Contacts> | ErrorResponse;
+
+
+
+@summary("List certificate issuers for a specified key vault.")
+/**
+* The GetCertificateIssuers operation returns the set of certificate issuer resources in the specified key vault. This operation requires the certificates/manageissuers/getissuers permission.
+*/
+@route("/certificates/issuers")
+@get op getCertificateIssuers(
+        ...ApiVersionParameter,
+        /**
+        * Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+        */
+        @maxValue(25)
+        @minValue(1)
+        @query("maxresults")
+        "maxresults?": int32
+): ArmResponse<CertificateIssuerListResult> | ErrorResponse;
+
+
+
+@summary("Sets the specified certificate issuer.")
+/**
+* The SetCertificateIssuer operation adds or updates the specified certificate issuer. This operation requires the certificates/setissuers permission.
+*/
+@route("/certificates/issuers/{issuer-name}")
+@put op setCertificateIssuer(
+        ...ApiVersionParameter,
+        /**
+        * The name of the issuer. The value you provide may be copied globally for the purpose of running the service. The value provided should not include personally identifiable or sensitive information.
+        */
+        @path
+        "issuer-name": string,
+        /**
+        * Certificate issuer set parameter.
+        */
+        @body
+        @flattenProperty
```

Comment:
TypeSpec validation is upset about these `@flattenProperty` decorators, but after talking about this with Laurent, they seem to be necessary for KV. Is there a suppression or change we should make here?